content/en/actions/datastores/auth.md

```diff
@@ -65,8 +64,8 @@ To elevate your access:
 {{< partial name="whats-next/whats-next.html" >}}
 
 [1]: /account_management/rbac/permissions/?tab=ui#app-builder--workflow-automation
-[2]: https://app.datadoghq.com/app-builder/datastore
+[2]: https://app.datadoghq.com/actions/datastores
 [3]: /actions/workflows
 [4]: /actions/app_builder
 [5]: /service_management/app_builder/connections/
-[6]: https://app.datadoghq.com/app-builder/datastore
\ No newline at end of file
+[6]: https://app.datadoghq.com/actions/datastores
```

Comment:
Just noting that links 2 and 6 currently point to the same URL. This is perfectly fine and won't break anything, but you can also have multiple links reference the same numbered destination link. For example, both could point to [2].